source/agora/node/Node.d

```diff
@@ -110,8 +111,29 @@ public class Node : API
 
         this.config = config;
         this.taskman = this.getTaskManager();
+
+        // build the list of required quorum peers to connect to if we're a Validator
+        Set!PublicKey required_peer_keys;
+        if (this.config.node.is_validator)
```

Comment:
If `this.config.node.is_validator` is `false`, a unit test results in an error.

`Module tests failed: agora.test.NetworkDiscovery
core.exception.AssertError@source/agora/test/NetworkDiscovery.d(89): Node 0 has 2 peers: ["GAYDGXP436SZ7LV6B5EXRZX27TB2D2GQRFV3SRN775RU6OQPMJT347F5", "GDD62CVWBKJE4NXYMELIV6GHZ5AKJ2TNY6MFASHUQY24RFAKJ4EZHCUF"]`

Reply:
Where are you setting it to false? Can you show me this test-case?